examples/ltimes.cpp

```diff
@@ -312,19 +313,22 @@ int main(int RAJA_UNUSED_ARG(argc), char **RAJA_UNUSED_ARG(argv[]))
   constexpr size_t tile_z = 256;
   constexpr size_t tile_g = 0;
 
+  using RAJA::statement::Param;
+
   using EXECPOL = 
     RAJA::KernelPolicy<
 
+      // Create memory tiles
+      statement::InitLocalMem<RAJA::cpu_tile_mem, RAJA::ParamList<0,1,2>,
+
```

Comment:
Shouldn't this go after a set of loops?

Reply:
not in this case, this is just sequential loops with some fixed array work buffers